e2e-tests/package.json

```diff
@@ -8,7 +8,7 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@playwright/test": "^1.42.1",
+    "@playwright/test": "^1.52.0",
```

Comment:
Would you mind separating this in to it's own commit?

This should make debugging later easier if needed.